docs/docs/etl-pipeline-tutorial/5-automate-your-pipeline.md

```diff
@@ -10,22 +10,50 @@ There are several ways to automate assets [in Dagster](/guides/automate). In thi
+Cron-based schedules are common in data orchestration. For our pipeline, assume that updated CSVs are uploaded to a file location at a specific time everyday by an external process.
 
-If we look back at the `joined_data` asset we defined earlier we can now discuss the `automation_condition` within the `dg.asset` decorator. This is [declarative automation](/guides/automate/declarative-automation), which understands the status of an asset and all of its dependencies. We can use declarative automation for a number of different workflows. In this case we can attach a schedule directly to the asset. Now everyday at midnight, the `joined_data` will execute.
+We can include this schedule information within the `dg.asset` decorator. This is [declarative automation](/guides/automate/declarative-automation), which understands the status of an asset and all of its dependencies. We can use declarative automation for a number of different workflows. In this case we can attach a schedule directly to the asset. Now everyday at midnight, our assets will execute.
 
-This is just one way we can use declarative automation.
+<CodeExample
+  path="docs_snippets/docs_snippets/guides/tutorials/etl_tutorial/src/etl_tutorial/defs/assets.py"
+  language="python"
+  startAfter="start_ingest_assets_3"
+  endBefore="end_ingest_assets_3"
+  title="src/etl_tutorial/defs/assets.py"
+/>
 
 ## 2. Other asset automation
 
 Now let's look at `monthly_sales_performance`. This asset should be executed once a month, but setting up an independent monthly schedule for this asset isn't exactly what we want -- if we do it naively, then this asset will execute exactly on the month boundary before the last week's data has had a chance to complete. We could delay the monthly schedule by a couple of hours to give the upstream assets a chance to finish, but what if the upstream computation fails or takes too long to complete?
 
-We already set this in the `monthly_sales_performance` and `product_performance` by setting the `automation_condition`. We want it to update when all the dependencies are updated. To accomplish this, we will use the `eager` automation condition.
+We already set this in the `monthly_sales_performance` by setting the `automation_condition`. We want it to update when all the dependencies are updated. To accomplish this, we will use the `eager` automation condition.
+
+<CodeExample
+  path="docs_snippets/docs_snippets/guides/tutorials/etl_tutorial/src/etl_tutorial/defs/assets.py"
+  language="python"
+  startAfter="start_monthly_sales_performance_asset_highlight"
+  endBefore="end_monthly_sales_performance_asset_highlight"
+  title="src/etl_tutorial/defs/assets.py"
+/>
+
+# 3. Enabling automation
+
+With declarative automation set for our assets, we can now enable the automation condition.
+
```

Comment:
Again, we'll need to tell people to run `dg dev` and navigate to the UI, then do the following steps